lib/data_update_scripts/20210119131811_replace_instances_of_banned_and_comment_banned.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class ReplaceInstancesOfBannedAndCommentBanned
+    def run
+      # Update names for the banned and comment_banned roles
+      # Details: https://github.com/forem/forem/pull/11581
+      Role.find_by(name: "banned").update(name: "suspended")
```

Comment:
I don't have banned users locally, so running this is a Rails console leads to an error:

```ruby
[3] forem(main)> Role.find_by(name: "banned").update(name: "suspended")
  Role Load (0.5ms)  SELECT "roles".* FROM "roles" WHERE "roles"."name" = $1 LIMIT $2  [["name", "banned"], ["LIMIT", 1]]
NoMethodError: undefined method `update' for nil:NilClass
from (pry):3:in `<main>'
```

I then ran the data update script and checked, its status is `failed`.

We should guard against this:

```suggestion
      Role.find_by(name: "banned")&.update(name: "suspended")
```

Reply:
this is likely going to break still